HUBER/iocBoot/iocHUBER-IOC-01/st-motor.cmd

```diff
@@ -5,12 +5,24 @@ set_pass0_restoreFile("$(IOCNAME)_built_settings.sav")
 set_pass1_restoreFile("$(IOCNAME)_built_settings.sav")
 
 $(IFDEVSIM) drvAsynIPPortConfigure("Huber$(MTRCTRL)", "localhost:$(EMULATOR_PORT)")
-$(IFNOTSIM) drvAsynIPPortConfigure("Huber$(MTRCTRL)", "$(HUBERADDR01)")
+$(IFNOTDEVSIM) $(IFNOTRECSIM) drvAsynSerialPortConfigure("Huber$(MTRCTRL)", "$(PORT)", 0, 0, 0)
+$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("Huber$(MTRCTRL)", -1, "parity", "$(PARITY=None)")
+$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("Huber$(MTRCTRL)", -1, "baud", "$(BAUD=19200)")
+$(IFNOTDEVSIM) $(IFNOTRECSIM) asynSetOption("Huber$(MTRCTRL)", -1, "bits", "$(BITS)")
```

Comment:
i'd keep these as defaulted as they are in config.xml - config.xml doesn't set the macro default, just indicates what it will be